UlKit/UlKit/OrderViewController.swift

```diff
@@ -0,0 +1,153 @@
+// OrderViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Описывает экран оплаты заказа
+class OrderViewController: UIViewController {
+    // MARK: - Public Properties
+
+    var orderItems: [(String, Int)]?
+    var orderSum: Int?
+
+    // MARK: - Private Properties
+
+    private var labels: [UILabel] = []
+    private var labelWidth: CGFloat = 280
+    private var labelHeight: CGFloat = 30
+    private var offsetSubview: CGFloat = 155
+
+    private lazy var closesButton: UIButton = {
+        let image = UIImage(named: "clear")
+        let button = UIButton()
+        button.setImage(image, for: .normal)
+        button.addTarget(self, action: #selector(closesViewController), for: .touchUpInside)
+        return button
+    }()
+
+    private var leftPatternImageView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.image = UIImage(named: "leftPattern")
+        return imageView
+    }()
+
+    private var rightPatternImageView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.image = UIImage(named: "rightPattern")
+        return imageView
+    }()
+
+    private var orderLabel: UILabel = {
+        let label = UILabel()
+        label.text = "Вашъ Заказъ"
+        label.font = .verdanaBold18()
+        label.textColor = .black
+        return label
+    }()
+
+    private var finalCostLabel: UILabel = {
+        let label = UILabel()
+        label.font = .verdanaBold18()
+        label.textColor = .black
+        return label
+    }()
+
+    private var patternLogoImageView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.image = UIImage(named: "menu_pic")
+        return imageView
+    }()
+
+    private lazy var payButton: CustomButton = {
+        let button = CustomButton(type: .system)
+        button.setTitle("Оплатить", for: .normal)
+        button.titleLabel?.font = UIFont(name: "Verdana-Bold", size: 18)
+        button.frame.origin.x = 15
+        button.setTitleColor(.white, for: .normal)
+        button.frame.origin.y = 600
```

Comment:
можно в одну строчку
```suggestion
        button.frame.origin = CGPoint(x: 15, y: 600)
```